xml/System.Threading/Timer.xml

```diff
@@ -481,7 +481,7 @@ Sub TimerCallback(state As Object)
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="dueTime" /> or <paramref name="period" /> parameter is negative and is not equal to <see cref="F:System.Threading.Timeout.Infinite" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="dueTime" /> or <paramref name="period" /> parameter is negative and is not equal to (-1).</exception>
```

Comment:
Why change this? Isn't the goal to standardize on the members of Timeout?

Reply:
Absolutely right, bad replace.

Reply:
It's -1 in my copy?

Reply:
I didn't even change that thought I don't think. weird.